modules/project-docs/pages/compatibility.adoc

```diff
@@ -28,7 +28,7 @@ It is best to upgrade either the SDK or the Couchbase version you are using.
 [#table_sdk_versions]
 [cols="40,25,20,20,25"]
```

Comment:
How about [cols="30,25,20,20,35"] -to make room for the 
*◎* _(see below)_

Reply:
yeah, you mentioned expanding the column - I noticed that because the header "SDK 3.0, 3.1" is already fairly long, it "looked fine on my device"...

(e.g. I'm not sure how best to test what the optimal width is? Happy to go with 35 and your intuition of course.)

Reply:
That was intuition, I didn't have time to test - but one of us will have to before it reaches prod ;)

Reply:
OK, in that case, let's say my local test is Good Enough For Staging, and we'll see there from a few different browsers if it needs tweaking?